ecl/hqlcpp/hqlecl.cpp

```diff
@@ -608,6 +609,10 @@ void HqlDllGenerator::addWorkUnitAsResource()
     code->addCompressResource("WORKUNIT", wuXML.length(), wuXML.str(), NULL, 1000);
 }
 
+void HqlDllGenerator::addArchiveAsResource(StringBuffer &buf)
+{
+    code->addCompressResource("ARCHIVE", buf.length(), buf.str(), nullptr, 1000);
```

Comment:
I think the id is meant to be unique.  And need to be careful not to clash with the manifest resources (see MANIFEST_BASE).  I think it is required to unique for windows, I can't remember how linux works.

Reply:
MANIFEST and WORKUNIT already both use 1000 as far as I can see.

I think it needs to be unique for each name, but not across names (but I can't remember either)